<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_System</class>
 <widget class="QWidget" name="Login_System">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fire</string>
  </property>
  <widget class="QFrame" name="widgets_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>430</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(24, 24, 24);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="lineEdit_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="username_entry">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 4px solid rgb(45, 45, 45);
border-color:rgb(56, 56, 56);
background-color: rgb(30,30,30);</string>
     </property>
     <property name="placeholderText">
      <string>UserName:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_entry">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 4px solid rgb(45, 45, 45);
border-color:rgb(56, 56, 56);
background-color: rgb(30,30,30);</string>
     </property>
     <property name="placeholderText">
      <string>Password:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="login_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(56, 56, 56);
	border-radius: 25px;
}
</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Entypo_d83c(0)_128.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="credits">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 49, 49);</string>
    </property>
    <property name="text">
     <string>  Created By Ejiofor Celestine</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
